packages/server/src/scim/routes.test.ts

```diff
@@ -166,7 +166,7 @@ describe('SCIM Routes', () => {
     expect(patchResponse.body.active).toBe(false);
   });
 
-  test.skip('Create missing medplum user type', async () => {
+  test('Create, missing medplum user type, creates a Practitioner', async () => {
```

Comment:
is there a reason for this change in behavior? My preference would be to leave the existing behavior for missing user type as is. 

Because we don't yet _require_ Practitioners to have access policies, we want to avoid a misconfiguration leading to a user being given "Practitioner" status

Reply:
This actually already is the default behavior, I'm updating this test to match it. It changed in #4134 (see https://github.com/medplum/medplum/pull/4134/files#diff-e40593c8cfa0f88e9466ecbdde25367e42fde68e5dbf33fa4ab57e276881f420R199-R200).

That said, up until now if you called this endpoint without a user type, it would have created a Practitioner that had the default patient access policy applied (if one was defined). So there is potentially a regression if someone is depending on that behavior 🤔 

Any suggestions on the best thing to do here?

Reply:
Good question. Think best would be to throw the 400 as the original (skipped) test had it written. For this kind of stuff related to auth access, generally prefer everything to be super explicit vs. allowing implicit behavior